var/spack/repos/builtin/packages/py-reprozip/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyReprozip(PythonPackage):
+    """A tool simplifying creating reproducible experiments from CLI executions.
+
+    It tracks operating system calls and creates a bundle that contains all the
+    binaries, files and dependencies required to run a given command on the
+    author’s computational environment (packing step). A reviewer can then
+    extract the experiment in his environment to reproduce the results
+    (unpacking step).
+
+    This is the component responsible for the packing step on Linux
+    distributions.
+
+    Please refer to reprounzip, reprounzip-vagrant, and reprounzip-docker for
+    other components and plugins.
+
+    """
+
+    homepage = "https://www.reprozip.org/"
+
+    pypi = "reprozip/reprozip-1.2.tar.gz"
+
+    maintainers("charmoniumQ")
+
+    version("1.2", md5="a98b7f04c52c60072e3c42da21997d3ad41161ff6cb1139e18cda8d3012120f9")
+
+    def global_options(self, spec, prefix):
```

Comment:
Note that `--global-option` is deprecated in pip 23.1 and will eventually be removed like `--install-option` was. So for now this is fine, but eventually we'll have to limit which versions of pip are allowed. Is this really the recommended way to specify where sqlite lives? Are there any alternatives?